.github/workflows/deploy.yml

```diff
@@ -33,7 +33,9 @@ jobs:
         #   run: go test -v ./...
 
       - name: Clone documentation
-        run: git clone https://github.com/gazebosim/docs
+        run: |
+          git clone https://github.com/gazebosim/docs
+          (cd docs; git log -n 1)
```

Comment:
Would this interfere with the `zip` command below?

Reply:
Not because of the directory change, which is done in a subshell and thus doesn't affect the outer context.

To try it at home `pwd; (cd /tmp; pwd); pwd`.

Reply:
TIL subshells. Neat!